Assets/OKU/BulletController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletController : MonoBehaviour
+{
+    public int offset = 1;     //弾を発射する際にプレイヤーと被らないようにするためのオフセット
+    private GameObject player; //プレイヤーを取得するための変数
+    private Vector3 playerpos; //プレイヤーの座標
+    public GameObject BulletPrehab; //弾のPrehabを格納する変数
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        player = gameObject.transform.parent.gameObject; //親オブジェクトであるplayerを取得
```

Comment:
what is this?

Reply:
https://teratail.com/questions/205652

gameObject(child).transform.parent.gameObject(parent)
gameObject can be any name.